<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneEditWidget</class>
 <widget class="QWidget" name="SceneEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>591</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>581</height>
    </rect>
   </property>
   <property name="title">
    <string>Tools</string>
   </property>
   <widget class="QPushButton" name="brushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources.qrc">
      <normaloff>:/files/images/brush.png</normaloff>:/files/images/brush.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="brushSizeSlider">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Brush size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eraserButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources.qrc">
      <normaloff>:/files/images/eraser.png</normaloff>:/files/images/eraser.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="eraserSizeSlider">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Eraser size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colorButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources.qrc">
      <normaloff>:/files/images/color-palette.png</normaloff>:/files/images/color-palette.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="addTextButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::InsertText"/>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="addShapeButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::MediaPlaybackStop"/>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>27</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="addImageButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::ImageLoading"/>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="scaleSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>50</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="rotateSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mergeShapesButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Merge Shapes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveImageButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startMotionButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Motion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources.qrc">
      <normaloff>:/files/images/direct-selection.png</normaloff>:/files/images/direct-selection.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="clearCanvas">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditDelete"/>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="changeBackground">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/files/images/color-palette.png</normaloff>:/files/images/color-palette.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
